stress-test/pipeline.yaml

```diff
@@ -0,0 +1,144 @@
+##############################################################################
+# YAML Anchors
+##############################################################################
+infrastructure_vars: &infrastructure_vars
+  TF_VAR_gcp_folder_id: ((project.folder_id))
+  TF_VAR_gcp_billing_account: ((project.billing_account))
+  TF_VAR_project_env: ((project.label))
+  TF_VAR_machine_type: ((k8s.node_machine_type))
+  TF_VAR_project_id: ((project.id))
+
+benchmark_vars: &benchmark_vars
+  PROJECT_ID: ((project.id))
+  REGION: ((project.region))
+  REQUESTS_JSON: ((benchmark.requests_json))
+  LOCUST_OPTIONS: '--no-web --clients ((benchmark.clients_per_instance)) --hatch-rate ((benchmark.hatch_rate)) --run-time ((benchmark.run_time)) --csv=output -L WARNING'
+  RUNNER_FULLY_QUALIFIED_DOMAIN_NAME: ((benchmark.target_fully_qualified_domain_name))
+  DOCKER_REGISTRY: ((docker.app_registry))
+  IMAGE_TAG: ((benchmark.image_tag))
+  USER_WAIT_TIME_MIN_SECONDS: ((benchmark.user_wait_time_min_seconds))
+  USER_WAIT_TIME_MAX_SECONDS: ((benchmark.user_wait_time_max_seconds))
+  REQUESTED_CPU_PER_POD: ((k8s.available_cpu_per_pod))
+  REQUESTED_MEMORY_PER_POD: ((k8s.available_memory_per_pod))
```

Comment:
We should change these var in the benchmark report for consistency.

Reply:
Ah ok - ignore my comment above then :+1:

Reply:
https://github.com/ONSdigital/eq-survey-runner-benchmark/pull/16/commits/965e442d142b9e3ed03957851c9bab47fee751f2